recipes/scorep/9.0.rc2/mc/repo/packages/libunwind/package.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
As far as I can tell this definition is exactly what's in 0.23.1 in upstream spack. Is this needed, or am I missing some change you've made compared to upstream?

Reply:
```
diff ~/packages/libunwind/package.py -d 9.0/gh200/repo/packages/libunwind/package.py
54c54
<         values=any_combination_of("coredump", "ptrace", "setjump"),
---
>         values=any_combination_of("coredump", "ptrace", "setjmp"),
```